rest_framework-stubs/pagination.pyi

```diff
@@ -87,7 +87,7 @@ class CursorPagination(BasePagination):
     cursor_query_description: str = ...
     page_size: int = ...
     invalid_cursor_message: str = ...
-    ordering: str = ...
+    ordering: Union[str, List[str, ...], Tuple[str, ...]] = ...
```

Comment:
It should be `Union[str, List[str], Tuple[str, ...]]`
Thanks for the fix!

Reply:
Yes... Fingers being quicker than brain... 🚀

Reply:
Can't it just be `Sequence[str]`?

Reply:
Yeah, it would work I think, except that there is this `assert` in the source that explicitly checks for these 3 types.

Reply:
Yes, I agree with @frnhr here. There's an explicit runtime typecheck there.